start.rst

```diff
@@ -2,16 +2,26 @@
 
 
 Introduction
-=============
+============
 
 NorESM2 User's Guide
 ^^^^^^^^^^^^^^^^^^^^
-
 This guide instructs both novice and experienced users on building and running NorESM2 for various experiment set ups. The chapters attempt to provide relatively detailed information on how to make, set up, build, run and modify experiments using NorESM2.
 
 
+NorESM2.1
+^^^^^^^^^
+NorESM2.1 is a technical development version of NorESM2 that includes a few bug fixes along with numerous code improvements from the NorESM2.0 release (NorESM tag release-noresm2.0.6). This implies that NorESM2.1 rely on the same make, set up, build and run procedures as NorESM2.0, but will not generally produce the same model output. We will refer to NorESM2.1 as "NorESM2" throughout this documentation.
```

Comment:
rely -> relies & suggest compset creation instead of make:
```suggestion
NorESM2.1 is a technical development version of NorESM2 that includes a few bug fixes along with numerous code improvements from the NorESM2.0 release (NorESM tag release-noresm2.0.6). This implies that NorESM2.1 relies on the same compset creation, set up, build and run procedures as NorESM2.0, but will not generally produce the same model output. We will refer to NorESM2.1 as "NorESM2" throughout this documentation.
```

Reply:
Thanks! I have updated the PR.

Reply:
@gold2718 - I thought I had updated the PR with your suggestion, and it shows up in the commits list, but it doesn't look like the file has changed. Any idea why this would happen? I don't understand this.

Reply:
I have made a new commit with the same changes, so now it shows up in the `start.rst`file

Reply:
@TomasTorsvik, it looks to me like you made the change (commit d316906) but then reverted that change (commit cbcdb19). I am seeing the original version here.

Reply:
@gold2718 - Sorry, I got confused about the updates. I reset the branch head to `d316906`